www/_posts/2017-04-21-cordova-7.md

```diff
@@ -0,0 +1,46 @@
+    name: Audrey So
+    url: https://twitter.com/aud_rey_so
+title:  "Apache Cordova 7.0.0"
+categories: news
+tags: news releases
+---
+
+We are happy to announce that `Apache Cordova 7.0.0` has been released!
+
+Most notable changes include:
+* If a `package.json` does not exist in your project, it will be auto-created for you when `cordova prepare` is called.
+* When adding a platform or plugin, it will automatically save that platform or plugin to your `config.xml` and `package.json`. Details about platform and plugin versions are also automatically saved in `config.xml` and `package.json`. The `--save` flag is no longer required to save. Use `--nosave` to prevent saving to `config.xml` or `package.json`.
+* Fetch is now the default setting for fetching platforms. Fetch uses your system `npm` to `npm install` modules into your project. The `--fetch` flag is no longer required. Use the `--nofetch` flag to use the older cordova fetching method (pre `cordova@7`) and prevent using `npm install` to do the fetching.
+* After `cordova prepare` is run, `package.json` and `config.xml` should contain identical platforms and versions.  In case of conflicts, `package.json` is given precedence over `config.xml`. For example, suppose `package.json` contains `cordova-android@6.0.0` and `config.xml` contains `cordova-android@4.0.0`. After `cordova prepare` is run, `config.xml` and `package.json` will each contain only `cordova-android@6.0.0`.
+* Users can install any platform due to removing known platform check for platform API. For instance, users can now add custom platforms to their projects.
+* Platforms and plugins are now required to have a package.json file.
+
+Example: 
+
+	cordova platform add custom-platform-name
+
+Link to how to create your own platforms:
+
+* A cordova config command has been created to `set`, `get`, `delete`, `edit`, and `list` global cordova options. For example, you can use the following command `cordova config set <key> <value>` to set the value of autosave or fetch to true or false. 
+
+In this case, you are turning off autosave as the default setting. Example: 
+
+	cordova config set autosave false
+
+In this case, you are turning off fetch as the default setting. Example:
+
+	cordova config set fetch false
+
+In addition, the cordova config command supports browserify setting, which allows the JavaScript of plugins to be loaded at build time compared to run time. For instance, if the browserify value is not explicility passed in by the user, the cordova config command will automatically set the browserify value saved in `~/.config/configstore/` to saved `globally`. Users can get and set browserify. Example:
+	
+	cordova config get browserify
+	cordova config set browserify true
+
+* Removed support for node 0.x. Only node versions 4.0 and up are supported.
+* Removed support for `cordova-windows@8` and under.
+* Removed support for `cordova-android@4` and under.
+* Removed support for `cordova-ios@3` and under.
```

Comment:
Hmm, is "and under" the right word to use when talking about version numbers? Maybe "and lower"?